terms/lossfunctions_ag.tex

```diff
@@ -0,0 +1,59 @@
+\chapter{Loss Functions}
+\label{ch:loss functions}\index{loss functions}
+
+The driving force of any mathematical optimization problem, especially in machine learning, is the loss function, measuring how well an algorithm is performing with regards to its target. In general terms, the evaluation for any learning problem is done by taking a predicted or estimated output and compare it to the relative real recorded outcome, typically in a dataset. The expected value of the loss is called the statistical risk. The formulation of the loss function is crucial and must be accurately defined according to the selected algorithm and the selected data for the particular optimization problem. The loss function must reproduce a numerical value that somehow reflects the level of achievement desired by the algorithm.  In machine learning we are typically attempting to find the parameters that fit our model and that minimizes the loss function. Minimizing the loss function is one of the fundamental foundations of statistical machine learning \citep{pmlr-v32-yanga14}. 
+
+\begin{marginfigure}%
+	\includegraphics[width=\linewidth]{loss_functions/cross_entropy.png}
+	\caption{Log Loss Function}
+	\label{fig:logloss}
+\end{marginfigure}
+
+
+The three canonical loss functions as described in Vapnik’s \citep{vapnik1998statistical} work are as follows:
```

Comment:
See point 26 in guidelines.  Did you see the pdf you generate?  Strange characters due to the single quote.